irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -15,8 +15,10 @@
  * limitations under the License.
  */
 
+#include "validation/impl/stateful_validator_impl.hpp"
```

Comment:
clang broken. This is the second PR where u fail with clang-format. Pls, just make it up.